src/Tribe/Integrations/WPML/Defaults.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__WPML__Defaults
+ *
+ * Handles sensible defaults for to The Events Calendar in WPML.
+ */
+class Tribe__Events__Integrations__WPML__Defaults {
+
+	/**
+	 * @var string The name of the sub-option that will store the first run flag.
+	 */
+	public $defaults_option_name = 'wpml_did_set_defaults';
+
+	/**
+	 * @var Tribe__Events__Integrations__WPML__Defaults
+	 */
+	protected static $instance;
+
+	/**
+	 * @var SitePress
+	 */
+	protected $sitepress;
+
+	/**
+	 * @var Tribe__Settings_Manager
+	 */
+	protected $settings_manager;
+
+	/**
+	 * Tribe__Events__Integrations__WPML__Defaults constructor.
+	 *
+	 * @param SitePress|null               $sitepress
+	 * @param Tribe__Settings_Manager|null $settings_manager
+	 */
+	public function __construct( SitePress $sitepress = null, Tribe__Settings_Manager $settings_manager = null ) {
```

Comment:
Public, again for reasons of testing and for the same reason above: if we want to prevent devs from misusing the code we would end up locking it up in private and final methods and classes. Which is a thing that makes working with it almost impossible